src/cursor/cur_backup_incr.c

```diff
@@ -41,6 +41,9 @@ __curbackup_incr_blkmod(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_CURSOR_BAC
     WT_ASSERT(session, cb->incr_src != NULL);
 
     WT_RET(__wt_metadata_search(session, btree->dhandle->name, &config));
+    ret = __wt_config_getones(session, config, "checkpoint", &v);
+    cb->has_ckpt = ret == WT_NOTFOUND;
```

Comment:
Did you manage to trigger the WT_NOTFOUND path here? I'm always a bit uncertain about when WT_NOTFOUND will be returned (I think it will be in this case because the configuration string is coming from metadata, so doesn't have a default value built in, but wouldn't want to be quoted). Most usages also check for `v.len == 0`, not just `WT_NOTFOUND`
It would also be better to use `WT_ERR_NOTFOUND_OK(__wt_config_getones(session, config, "checkpoint", &v), true);` That will mean that other errors will be captured and returned.

Reply:
When looking into making the changes Sue suggested this morning, I realised that this code is incorrect. I'm going to put this into draft mode while I refactor.